Projects/Faker/Faker/Program.cs

```diff
@@ -1,63 +1,72 @@
 ﻿using System;
-using Faker;
-using Faker.Resources;
-using Newtonsoft.Json;
+using System.Linq;
+using System.Text.Json;
 
-namespace FakerDemo
-{    
-    internal class Program
+for (var i = 0; i < 25; i++)
+{
+    var user = CreateUser();
+
+    var userString = user.ToString();
+
+    // or, if you want JSON:
+    // var userString = JsonSerializer.Serialize(user, new JsonSerializerOptions {WriteIndented = true})
+
+    Console.WriteLine(userString);
+}
+
+User CreateUser()
+{
+    var fullNameInput = Faker.Name.FullName();
+
+    var titles = new[]
     {
-        class User
-        {
-            public string FullName { get; set; }
-            public string FirstName { get; set; }
-            public string LastName { get; set; }
-        }
-        static void Main(string[] args)
+        "Prof",
+        "Dr",
+        "Mr",
+        "Mrs",
+        "Miss",
+        "Ms",
+    };
+
+    var index = 0;
+    foreach (var title in titles.OrderByDescending(x => x.Length))
```

Comment:
So this iterates all the title array (dictionary) objects in title variable and checks if the fullNameInput does not starts with any of these?

Reply:
Kind of yes, it's checking if the fullname starts with any of the titles, starting with the ones with most characters (mrs vs mr), it stores the length of the title (plus optional dot) as the starting position of the name(s). If no title is found, index should still be 0.